lib/modules/versioning/distro.ts

```diff
@@ -35,12 +39,30 @@ export class DistroInfo {
       const schedule = this._distroInfo[version];
       this._codenameToVersion.set(schedule.series, { version, ...schedule });
     }
+
+    const arr = Object.keys(this._distroInfo).sort(
+      (a, b) => parseFloat(a) - parseInt(b)
```

Comment:
one float and one int? 🤔

Reply:
thats a typo, 
funny thing is that ive fixed this before. thats weird 🙃